kuma/users/views.py

```diff
@@ -516,7 +502,10 @@ def get_form(self, form_class):
                     label = _('%(email)s Unverified')
                 next_email = email_address['email']
                 choices.append((next_email, label % {'email': next_email}))
-            choices.append((form.other_email_value, _('Other:')))
+            if extra_email_addresses:
+                choices.append((form.other_email_value, _('Other:')))
+            else:
+                choices.append((form.other_email_value, _('Email:')))
```

Comment:
I think we should require that users signing up with github have at least one verified email, and present a message to them asking that they go verify their email with github in order to sign up for MDN if there aren't any. Ideally a user signing up with github should never type an email address into an MDN form.